src/_posts/languages/nodejs/2000-01-01-start.md

```diff
@@ -98,6 +98,21 @@ By default, dependencies present in the `devDependencies` field are installed. A
 
 The pruning step can be skipped by setting the environment variable `YARN_PRODUCTION` to any value.
 
+#### Skip Pruning
+
+If you need access to packages declared under devDependencies at runtime or in a different buildpack then, depending on the package manager used, you can set one of the following environment variables to skip the pruning step:
+
+| Package manager | Version | Environment variable    | Value              |
+|-----------------|---------|-------------------------|--------------------|
+| npm             | Any     | `NPM_CONFIG_PRODUCTION` | `true`  or `false` |
```

Comment:
issue: well, from the code sent by François, the value to skip the pruning should be `true`, shouldn't it?

```suggestion
| npm             | Any     | `NPM_CONFIG_PRODUCTION` | `true`  |
```